content/blog/rustyjam-02/index.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "RustyJam 2"
+date = 2022-06-11
+transparent = true
++++
+
+![Aaron: a drawing of a humanoid fox](aaron.png)
+
+Hey everyone, we're finally announcing the [RustyJam #2][itch]!
+It's a great chance to try out some new Rusty tech,
+form a team of like-minded rustaceans, and feel what
+a full cycle of making a game in Rust feels like in miniature!
+
+Submissions will be open from June 18th to June 26th.
+Voting on submissions will begin as soon as the jam ends and will run for a week.
+
+This jam focuses more on using Rust than anything else.
+That means you aren't restricted on your design, music, or graphics,
+as long as you use Rust to make it!
+
+The optional-to-use theme for the jam is going to be announced
+[in our Discord][discord] and pinned in the community section.
```

Comment:
Very pedantic point - 'our' Discord implies that this is a space that the WG manages/moderates, which I don't think is true (though there is overlap).

```suggestion
[in the Rusty Jam Discord][discord] and pinned in the community section.
```